man/mosh-server.1

```diff
@@ -82,6 +82,44 @@ Locale-related environment variable to try as part of a fallback
 environment, if the startup environment does not specify a character
 set of UTF-8.
 
+.SH ENVIRONMENT VARIABLES
+These variables allow server-side configuration of Mosh's behavior.
+They may be set by administrators in system login/rc files,
+/etc/login.conf, or similar mechanisms, or users in their shell's
+login/rc files.  \fBmosh-server\fP passes these variables to the login
+session and shell that it starts, but changing them there will have no
+effect.
+
+.TP
+.B MOSH_SERVER_NETWORK_TMOUT
+When set to an integer number, this variable specifies how long the
+server will wait, in seconds, to receive an update from the client
+before exiting.  Since \fPmosh\fP is very useful for mobile clients
+with intermittent operation and connectivity, we suggest setting this
+variable to a high value, such as 604800 (one week) or 2592000 (30
+days).  When not set, \fBmosh-server\fP will wait indefinitely for a
+client to reappear.  This variable is somewhat similar to the
+\fBTMOUT\fP variable found in many Bourne shells.  However, it is not
+a login-session inactivity timeout; it only applies to network
+connectivity.
+
+.TP
+.B MOSH_SERVER_NETWORK_TMOUT
+When set to an integer number, this variable specifies how long the
+server will wait, in seconds, to receive an update from the client
+before allowing \fBSIGUSR1\fP to terminate a session.  When unset,
+\fBSIGUSR1\fP will always terminate sessions.  With this feature,
+users and administrators may implement scripts to clean up
+disconnected Mosh sessions without fear of accidentally killing those
+that have active connectivity.  With this variable set, a user
```

Comment:
On a quick skim I mis-interpreted this as "only users connected via Mosh can [..]". Perhaps instead of "with this variable set, a user [..] can issue", say "Another use case is to allow a user [..] to issue".